src/main/webapp/app/shared/model/delusion.model.ts

```diff
@@ -0,0 +1,8 @@
+export interface Delusion {
+    key?: string;
+    label?: string;
+}
+
+export function copyDelusion(delusion: Delusion): Delusion {
```

Comment:
I can see you were trying to copy at some point - you were on the right track it seems